### ISSUE TITLE
Add  self-hosted newsblur

### ISSUE THREAD

**Issue by dumprop** [Ranchero-Software/NetNewsWire#2806]
https://github.com/samuelclay/NewsBlur
official ios client have custom domain support too
I think it will be helpful

**Comment by vincode-io** [comment]
Have you got a working self hosted NewsBlur installation? It looks pretty non-trivial to set up.

**Comment by dumprop** [comment]
No, but it looks interesting and probably I will set up it in near future (I want to test popular self hosted solutions)
So, seems like newsblur already implemented, only one config is needed
